pkgs/applications/misc/msglint/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation {
+  name = "msglint-1.04";
+
+  setSourceRoot = "sourceRoot=`pwd`";
+
+  src = fetchurl {
+    url = "https://tools.ietf.org/tools/msglint/msglint-1.04.tar.gz";
+    sha256 = "1yasj6nijip3z86m526bsssd5vgqrx233azvpz1c0z564wngvzbm";
+  };
+
+  patches = [ ./fixfree.patch ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp msglint $out/bin/msglint
+  '';
+
+  meta = with stdenv.lib; {
+    description  = "RFC 822/MIME/DSN/MDN/Tracking-Status message validator.";
+    homepage     = "https://tools.ietf.org/tools/msglint/";
+    downloadPage = "https://tools.ietf.org/tools/msglint/";
+    maintainers  = with maintainers; [ apeyroux ];
+    license      = licenses.unfree;
+    platforms    = [ "i686-linux" "x86_64-linux" "x86_64-darwin" ];
```

Comment:
On first sight, I do not see any platform limitations in the source code. Thus, I would omit the `platforms` entry.